aliquotmaf/annotators/gnomad.py

```diff
@@ -55,33 +41,7 @@ def __init__(self, scheme, refpath, refpattern):
         self.path = refpath
         self.file_template = refpattern
         self.chrom = None
-        self.chrom_list = {
-            'chr1',
-            'chr2',
-            'chr3',
-            'chr4',
-            'chr5',
-            'chr6',
-            'chr7',
-            'chr8',
-            'chr9',
-            'chr10',
-            'chr11',
-            'chr12',
-            'chr13',
-            'chr14',
-            'chr15',
-            'chr16',
-            'chr17',
-            'chr18',
-            'chr19',
-            'chr20',
-            'chr21',
-            'chr22',
-            'chrX',
-            'chrY',
-            'chrM',
-        }
+        self.chrom_list = CHROM_LIST
```

Comment:
[nitpicky]

I would handle global variables in a class slightly differently:

Option 1: Let the methods access it directly rather than saving it as a self variable.

So on line 93 it would be `if chrom in CHROM_LIST`.

Option 2: Inject it on init:

```python

class GnomAD:
    def __init__(self, ..., _chrom_list=CHROM_LIST):
        self.chrom_list = _chrom_list
```

In general option 2 is better for things that might change during testing while option 1 is for more immutable things (like the names of human chromosomes (hopefully immutable))